internal/pkg/cli/app_deploy.go

```diff
@@ -30,6 +31,7 @@ import (
 
 const (
 	inputImageTagPrompt = "Input an image tag value:"
+	addonsURLPattern    = "https://%s/%s"
```

Comment:
Is this used?

Reply:
😓

Reply:
It's good to keep a few of these in - helps make me feel like I did a good PR :P